assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/positive.json

```diff
@@ -0,0 +1,57 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "A sample template",
+  "Resources": {
+    "taskdefinition": {
```

Comment:
Change "taskdefinition" to "taskdefinition3". Sorry I missed this earlier.

Reply:
done